<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductKeyForm</class>
 <widget class="QWidget" name="ProductKeyForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ключ продукта.</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>220</y>
     <width>123</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhUppercaseOnly</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>WWWWW</string>
   </property>
   <property name="maxLength">
    <number>29</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>220</y>
     <width>123</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WWWWW</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>220</y>
     <width>123</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WWWWW</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>220</y>
     <width>123</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WWWWW</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>220</y>
     <width>123</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WWWWW</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:Times New Roman;
font-style: normal;
font-weight: bold;

font-size:26px;
color:#3A91E8;</string>
   </property>
   <property name="text">
    <string>Введите ключ продукта.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ключ продукта имеет следующий формат:  XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 

{

   border-style: solid;
 
   border-radius: 10;

    border-top          : 1px solid rgba(200, 200, 200, 255);

    border-left         : 1px solid rgba(200, 200, 200, 255);

    border-right        : 2px solid rgba(172, 172, 172);

    border-bottom       : 2px solid rgba(172, 172, 172);

   /*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(198, 198, 198), stop:1 rgba(225, 225, 225, 255));
*/
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 0.1 #eaeaea, stop: 0.49 #eaeaea, stop: 0.5 #eaeaea, stop: 1 #ffffff);
font-size: 16px;
}



QPushButton:hover 

{
/*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(163, 163, 163));
*/
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(196, 228, 255), stop:1 rgba(225, 225, 225, 255));
}



QPushButton:pressed 

{

    padding-top         : 2px;

    padding-left        : 2px;

    border-top          : 2px solid rgba(117, 117, 117, 255);

    border-left         : 2px solid rgba(117, 117, 117, 255);

    border-right        : 1px solid rgba(200, 200, 200, 255);

    border-bottom       : 1px solid rgba(200, 200, 200, 255);

    background-color    : rgba(172, 172, 172);

}



QPushButton:disabled 

{

    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 255, 255), stop:1 rgba(172, 172, 172));

}</string>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 

{

   border-style: solid;
 
   border-radius: 10;

    border-top          : 1px solid rgba(200, 200, 200, 255);

    border-left         : 1px solid rgba(200, 200, 200, 255);

    border-right        : 2px solid rgba(172, 172, 172);

    border-bottom       : 2px solid rgba(172, 172, 172);

   /*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(198, 198, 198), stop:1 rgba(225, 225, 225, 255));
*/
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 0.1 #eaeaea, stop: 0.49 #eaeaea, stop: 0.5 #eaeaea, stop: 1 #ffffff);
font-size: 16px;
}



QPushButton:hover 

{
/*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(163, 163, 163));
*/
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(196, 228, 255), stop:1 rgba(225, 225, 225, 255));
}



QPushButton:pressed 

{

    padding-top         : 2px;

    padding-left        : 2px;

    border-top          : 2px solid rgba(117, 117, 117, 255);

    border-left         : 2px solid rgba(117, 117, 117, 255);

    border-right        : 1px solid rgba(200, 200, 200, 255);

    border-bottom       : 1px solid rgba(200, 200, 200, 255);

    background-color    : rgba(172, 172, 172);

}



QPushButton:disabled 

{

    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 255, 255), stop:1 rgba(172, 172, 172));

}</string>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowError">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>671</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:Times New Roman;
font-style: normal;
font-weight: bold;

font-size:16px;
color:#FF0000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
